src/command_base.ts

```diff
@@ -53,7 +53,7 @@ export abstract class BaseCommand {
 	/**
 	 * Execute is the method called first when running a command.
 	 */
-	public abstract async execute(): Promise<Discord.Message | void>;
+	public abstract execute(): Promise<Discord.Message | void>;
```

Comment:
Do abstract methods just not need to be flagged async (I assume the return type being `Promise<Whatever>` handles that).

Reply:
Yes.